Formula/lib/libsidplayfp.rb

```diff
@@ -0,0 +1,40 @@
+class Libsidplayfp < Formula
+  desc "Library to play Commodore 64 music"
+  homepage "https://github.com/libsidplayfp/libsidplayfp"
+  url "https://github.com/libsidplayfp/libsidplayfp/releases/download/v2.12.0/libsidplayfp-2.12.0.tar.gz"
+  sha256 "bc4f4fa203dcf0736fe48c23dce9aa0db825370e5941e7595e4851efe6937cdc"
+  license "GPL-2.0-or-later"
+
+  head do
+    url "https://github.com/libsidplayfp/libsidplayfp.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "coreutils" => :build
+    depends_on "libtool" => :build
+    depends_on "xa" => :build
+  end
+
+  depends_on "pkgconf" => :build
+  depends_on "libgcrypt"
+
+  def install
+    system "autoreconf", "-vfi" if build.head?
+    system "./configure", "--disable-silent-rules", *std_configure_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~C
+      #include <stdio.h>
+      #include <sidplayfp/sidplayfp.h>
+      int main (void) {
+        sidplayfp play();
+        printf("%d.%d.%d", LIBSIDPLAYFP_VERSION_MAJ, LIBSIDPLAYFP_VERSION_MIN, LIBSIDPLAYFP_VERSION_LEV);
+        return 0;
+      }
+    C
+    system ENV.cxx, "test.cpp", "-L#{lib}", "-I#{include}", "-lsidplayfp", "-o", "test"
+    assert_equal version.to_s, shell_output("./test") unless head?
```

Comment:
```suggestion
    assert_equal version.to_s, shell_output("./test")
```

Reply:
The reason for the head check is because that test may fail for a head build. If the version gets bumped in a head commit before a new stable formula is released this version check test will fail, but I don’t think that would be a legitimate failure.

Reply:
I think the test is still helpful for checking the stable build (which would be run by CI)

but for the head build, since it would be constantly changing, that is the main reason why we dont do the `brew test` check as long as it builds out fine.

Reply:
Ahh I see. I thought I had seen a similar check elsewhere, but I’m not even sure where!

Reply:
np